### DISCUSSION TITLE
How to make siderbar tabs work when using trailingSlash:true?

### DISCUSSION THREAD

**Discussion by adyingdeath** [fuma-nama/fumadocs#1196] [Q&A]
I am using static export mode. And If I don't add a "trailingSlash:true" to the next config, the output folder will be like:
```
out/
├── docs.html
└── docs/
    └── page1.html
```

However, I need a "index.html" to deploy it. It should be like:
```
out/
└── docs/
    ├── index.html
    └── page1.html
```

This can be achieved by adding "trailingSlash:true" to the next config. However, when I add this, my sidebar tabs will not work, and the root folders will not work too.
For example, I have two root folders "A" and "B". Normally, If I am reading an article in "A", the pages in "B" will not display in the sidebar. However, when "trailingSlash:true", there will be B's pages even when I am reading A's pages and the sidebar tabs will not work as well. I am thinking that it may be because "trailingSlash:true" have change the path(adding a "/" to its tail). I've seen somewhere that fumadocs's official docs site is also deployed in static mode. How do you solve this?
Is there another way where I can output correct structures without adding a "trailingSlash:true"?

**Comment by fuma-nama** [comment]
It is probably a bug or misconfiguration of dynamic route `app/docs/[[...slug]]`, Fumadocs do not handle that. You can also specify the items via `sidebar.tabs` in DocsLayout, if you want to bypass the bug.

In the past we simply use `app/docs/[...slug]` because we have a `app/docs/page.tsx` page that's not written in MDX.

**Reply by adyingdeath** [reply]
What is the format? I've write something like this:
```
<DocsLayout
    tree={source.pageTree}
    sidebar={{
      tabs: [
        {
          title: "A",
          url: "/a",
        },
        {
          title: "B",
          url: "/b",
        },
      ],
      banner: (<Banner />)
    }}
    {...baseOptions}>
      {children}
    </DocsLayout>
```
And it seems not working.

**Reply by adyingdeath** [reply]
I give up solving this. It seems a bit tricky. I've opted for another compromise solution. Thank you!
